sbt-cross/src/main/scala/sbtcross/SbtCrossPlugin.scala

```diff
@@ -0,0 +1,31 @@
+package sbtcross
+
+import sbt._
+import StringUtilities.nonEmpty
+
+import scala.language.implicitConversions
+
+object SbtCrossPlugin extends AutoPlugin {
+  val autoImport = AutoImport
+  object AutoImport extends JvmCross with CrossKeys {
+    final implicit def toScalaJSGroupID(groupID: String) = null
+
+    final implicit def toCrossGroupID(groupID: String): CrossGroupID = {
+      nonEmpty(groupID, "Group ID")
+      new CrossGroupID(groupID)
+    }
+
+    final implicit def crossProjectFromBuilder2(
+        builder: CrossProject.Builder): CrossProject = {
+      builder.crossType(CrossType.Full)
+    }
```

Comment:
I suggest putting this implicit conversion in the companion object of `CrossProject.Builder` instead.